includes/admin/class-wcmp-export.php

```diff
@@ -334,9 +337,19 @@ public function add_shipments(array $order_ids, bool $process)
 
             $extraOptions = WCX_Order::get_meta($order, WCMYPA_Admin::META_SHIPMENT_OPTIONS_EXTRA);
             $colloAmount  = $extraOptions["collo_amount"] ?? 1;
+            $isMailboxPackage  = 'mailbox' === $this->getPackageTypeFromOrder($order);
+            $isPackage  = 'package' === $this->getPackageTypeFromOrder($order);
+            $isMultiColloCountry = in_array($order->get_shipping_country(), [self::COUNTRY_CODE_NL, self::COUNTRY_CODE_BE]);
 
             if ($colloAmount > 1) {
-                $collection->addMultiCollo($consignment, $colloAmount);
+                if ($isMultiColloCountry && $isPackage) {
```

Comment:
Deze logica zat er eerst niet in, moet dit echt zo werken? Mag je wel fake multicollo naar ieder land? Waarom doen we dan niet ook fake multicollo voor packages naar niet-multicollo landen?

Reply:
Je hebt nu weer 2x dezelfde conditie maar dan andersom (`$isMultiColloCountry`), de logica wordt nu onduidelijk, ik zie dit liever in een `else`. 

Weet je zeker dat het correct is dat er voor iedere package type fake-multicollo kan worden aangemaakt? Heb je gekeken hoe het in de oude versie werkte?